orchestrators/ProcurementOperations/tests/Unit/Coordinators/VendorRecommendationCoordinatorTest.php

```
@@ -0,0 +1,146 @@
+
+    public function enrich(VendorRecommendationRequest $request, array $candidates): array
+    {
+        return $this->enrichments;
+    }
+}
+
+final readonly class FixedRecommendationClock implements \Nexus\Common\Contracts\ClockInterface
+{
+    public function now(): \DateTimeImmutable
+    {
+        return new \DateTimeImmutable('2026-04-22T00:00:00Z');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '144', column '20'). (undefined)

(MissingImport)